lib/menu.ts

```diff
@@ -233,27 +261,32 @@ export function menuItemSubmenu(submenu: MenuCreateFunc, ...args: DomElementArg[
 
 export const cssMenu = styled('ul', `
   position: absolute;
-  background: white;
-  color: #1D1729;
-  min-width: 160px;
   outline: none;
-  border-radius: 3px;
-  box-shadow: 0 0 2px rgba(0,0,0,0.5);
   list-style: none;
-  padding: 16px 0px;
   margin: 2px;
   text-align: left;
+
+  font-size:     var(--weaseljs-font-size, 13px);
```

Comment:
all these variables are what's unnecessary IMO, setting `--weaseljs-min-width` has the same effect as setting `min-width`, so I see no benefit to writing the extra lines code and making the user learn the extra names.